.github/workflows/joystream-cli.yml

```
@@ -14,14 +14,14 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: build
+    - name: check
       run: |
         yarn install --frozen-lockfile
         yarn madge --circular types/
-        yarn workspace joystream-cli build
+        yarn workspace joystream-cli check
```

```suggestion
        yarn workspace joystream-cli checks
```

Its actually `checks`
`check` is a yarn [builtin command](https://classic.yarnpkg.com/en/docs/cli/check/#toc-yarn-check) that does sanity check on package.json and yarn.lock

---

sorry seems I missed the fix in later commit